src/pages/settings/gateways/common/hooks/useGatewayUtilities.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { useCompanyChanges } from '$app/common/hooks/useCompanyChanges';
+import { CompanyGateway } from '$app/common/interfaces/company-gateway';
+import { useCompanyGatewaysQuery } from '$app/common/queries/company-gateways';
+import { updateChanges } from '$app/common/stores/slices/company-users';
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+interface Params {
+  currentGateways: CompanyGateway[];
+  setCurrentGateways: Dispatch<SetStateAction<CompanyGateway[]>>;
+}
+export const useGatewayUtilities = (params: Params) => {
```

Comment:
`function` style of declaration, please.

Reply:
Changed!